openshift/ci-operator/static-images/receiver/Dockerfile

```diff
@@ -14,10 +14,12 @@
 # limitations under the License.
 #
 
-FROM quay.io/openshift-knative/17-jdk-centos7 as builder
+FROM registry.access.redhat.com/ubi9/openjdk-21 as builder
```

Comment:
I'd think we stay on `ubi8` here, since product is using RHEL-8 atm  (so we have less diff towards that)

Reply:
Makes sense. I didn't know they are still on ubi8. Updated it